### PR TITLE
fix(aws-amplify-vue): Change the event name to signIn

### LINKED ISSUE #2935: aws-amplify-vue: signOut event fired when cancelling signUp
**Describe the bug**
I'm using the <Authenticator> component and when pressing the "Create an account" link it fires the `signUp` event as expected. But when I then click the "Sign In" link on that page the `signedOut` event triggers. This is the same event that happens when clicking the "Sign out" button.

So if I listen to the `signOut` event and redirect a user to my home page or login page, this also happens when cancelling a user sign up. I don't think this is intended.

**Expected behavior**
I expect some sort of `cancelSignUp` event to fire instead of the `signedOut`. I believe the `signedOut` should only be fired when the user actually logs out.

**Desktop (please complete the following information):**
 - OS: Linux
 - Chrome 73

**Additional context**
aws-amplify 1.1.22
aws-amplify-vue 0.2.7

### PR DESCRIPTION
**Pull request by powerful23**
*Issue #, if available:*
fixes #2935 
*Description of changes:*

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by powerful23** [comment]
Need testing.

File: packages/aws-amplify-vue/src/components/authenticator/Authenticator.vue [hunk 1]
```diff
@@ -78,7 +78,7 @@ export default {
   methods: {
     updateDisplayMap: state => {
       return {
-        showSignIn: state === 'signedOut',
+        showSignIn: state === 'signedOut' || 'signIn',
```
Comment (alebiavati) [inline packages/aws-amplify-vue/src/components/authenticator/Authenticator.vue]: Please change the readme file as well

File: packages/aws-amplify-vue/src/components/authenticator/ConfirmSignUp.vue [hunk 2]
```diff
@@ -90,7 +90,7 @@ export default {
             .catch(e => this.setError(e));
     },
     signIn() {
-        AmplifyEventBus.$emit('authState', 'signedOut')
+        AmplifyEventBus.$emit('authState', 'signIn')
```
Comment (alebiavati) [inline packages/aws-amplify-vue/src/components/authenticator/ConfirmSignUp.vue]: the `ConfirmSignUp` component emits the `signedOut` event also after confirmation, we should change the event there too.

File: packages/aws-amplify-vue/src/components/authenticator/ForgotPassword.vue [hunk 3]
```diff
@@ -92,7 +92,7 @@ export default {
         .catch(e => this.setError(e));
     },
     signIn: function() {
-      AmplifyEventBus.$emit('authState', 'signedOut');
+      AmplifyEventBus.$emit('authState', 'signIn');
```
Comment (alebiavati) [inline packages/aws-amplify-vue/src/components/authenticator/ForgotPassword.vue]: the `ForgotPassword` component emits the `signedOut` event also after verification success, we should change the event there too.

File: packages/aws-amplify-vue/src/components/authenticator/SignUp.vue [hunk 4]
```diff
@@ -239,7 +239,7 @@ export default {
       return invalids.length < 1;
     },
     signIn: function() {
-      AmplifyEventBus.$emit('authState', 'signedOut')
+      AmplifyEventBus.$emit('authState', 'signIn')
```
Comment (alebiavati) [inline packages/aws-amplify-vue/src/components/authenticator/SignUp.vue]: the `SignUp` component emits the `signedOut` event also after `signUp` method success, we should change the event there too.

File: packages/aws-amplify-vue/src/components/authenticator/ConfirmSignUp.vue (hunk 2 above)
Comment (powerful23) [reply packages/aws-amplify-vue/src/components/authenticator/ConfirmSignUp.vue]: Nice catch!

File: packages/aws-amplify-vue/src/components/authenticator/ForgotPassword.vue (hunk 3 above)
Comment (powerful23) [reply packages/aws-amplify-vue/src/components/authenticator/ForgotPassword.vue]: Nice catch!

File: packages/aws-amplify-vue/src/components/authenticator/SignUp.vue (hunk 4 above)
Comment (powerful23) [reply packages/aws-amplify-vue/src/components/authenticator/SignUp.vue]: Nice catch!

File: packages/aws-amplify-vue/src/components/authenticator/Authenticator.vue (hunk 1 above)
Comment (powerful23) [reply packages/aws-amplify-vue/src/components/authenticator/Authenticator.vue]: Doc changed: https://github.com/aws-amplify/docs/pull/584

**Comment by powerful23** [comment]
> Need testing.

Tested with amplify vue sample.

File: packages/aws-amplify-vue/src/components/authenticator/SignUp.vue [hunk 5]
```diff
@@ -217,9 +217,9 @@ export default {
               this.logger.info('sign up success');
               AmplifyEventBus.$emit('localUser', data.user)
               if (data.userConfirmed === false){
-                return AmplifyEventBus.$emit('authState', 'confirmSignUp');
+                return AmplifyEventBus.$emit('authState', 'confirmSignUp');In
```
Comment (alebiavati) [inline packages/aws-amplify-vue/src/components/authenticator/SignUp.vue]: there's an extra `In` at the end of the line :)

File: packages/aws-amplify-vue/src/components/authenticator/Authenticator.vue (hunk 1 above)
Comment (alebiavati) [reply packages/aws-amplify-vue/src/components/authenticator/Authenticator.vue]: I think you need to update the README.md file in this repository as well.

File: packages/aws-amplify-vue/src/components/authenticator/SignUp.vue (hunk 5 above)
Comment (powerful23) [reply packages/aws-amplify-vue/src/components/authenticator/SignUp.vue]: Good catch!

**Pull request merged**